src/LiveApi.js

```diff
@@ -13,6 +13,8 @@ let WebSocket = typeof window !== 'undefined' ? window.WebSocket : MockWebSocket
 const shouldIgnoreError = error =>
     error.message.includes('You are already subscribed to') ||
         error.message.includes('Input validation failed: forget');
+const shouldNotThrow = json => 
+    json.error.code === 'SelfExclusion' && json.msg_type === 'authorize';
 
```

Comment:
Also, keep in mind users not familiar with the library will be surprised by this behaviour. At least currently this is a rather thin layer of abstraction.